actuator/src/main/java/org/tron/core/utils/TransactionUtil.java

```diff
@@ -305,4 +328,25 @@ public static long estimateConsumeBandWidthSize(final AccountCapsule ownerCapsul
     return DELEGATE_COST_BASE_SIZE + addSize;
   }
```

Comment:
Is this method already deprecated? If so, it is recommended to delete it to avoid confusion.